### DISCUSSION TITLE
Downgrade Zigbee2MQTT

### DISCUSSION THREAD

**Discussion by unxplained** [tteck/Proxmox#2269] [Help Needed]
Hi,

I'm facing similar issue to [this](https://github.com/Koenkk/zigbee2mqtt/issues/19747). Was wondering is there a way to downgrade my zigbee2mqtt version to 1.33.0 to see if it fixes my issue.

Thanks

**Comment by Vakrehus** [comment]
I think the quickest option would be to restore your backup.

**Reply by unxplained** [reply]
Don't have a backup that old sadly

**Comment by tteck** [comment]
Are you using a SkyConnect or another device from Silicon Labs, specifically from the EFR32MG2x/MGM21x and EFR32MG1x/MGM1x series, as a coordinator?

[These are experimental, don't use if you want a stable setup](https://www.zigbee2mqtt.io/guide/adapters/#experimental)

**Reply by unxplained** [reply]
yea, im using a an EFR32mg21 device. planning to buy another device but thinking to downgrade my zigbee2mqtt version in the meantime
